nodes/CIPP/CIPP.node.ts

```diff
@@ -40,12 +41,14 @@ export class CIPP implements INodeType {
 					{ name: 'GDAP', value: 'gdap' },
 					{ name: 'Tenant', value: 'tenant' },
 					{ name: 'User', value: 'user' },
+					{ name: 'CIPP', value: 'cipp' },
```

Comment:
Please put the resource into alphabetical order and then I can merge this (part of the n8n code quality guidelines)